webpack.config.js

```diff
@@ -5,21 +5,26 @@ const CleanCSSPlugin = require("less-plugin-clean-css");
 
 module.exports = {
     entry: {
-        demo: path.join(__dirname, "src/js/demo/index.js"),
-        parser: path.join(__dirname, "src/js/parser/index.js"),
-        main: path.join(__dirname, "src/js/main/index.js")
+        demo: path.resolve(__dirname, "src/js/demo/index.js"),
+        main: path.resolve(__dirname, "src/js/main/index.js")
     },
     output: {
-        path: path.join(__dirname, "_site/assets/js"),
+        path: path.resolve(__dirname, "_site/assets/js"),
         filename: "[name].js"
     },
     module: {
         rules: [
             {
                 test: /\.jsx?$/,
-                loaders: ["babel-loader"],
-                include: path.join(__dirname, "src/js"),
-                exclude: /node_modules|src\/js\/vendor/
+                loader: "babel-loader",
+                include: path.resolve(__dirname, "src/js"),
+                exclude: [
+                    path.resolve(__dirname, "node_modules"),
+                    path.resolve(__dirname, "src/js/demo/node_modules/lodash")
```

Comment:
From https://github.com/eslint/eslint/blob/master/webpack.config.js#L23

Reply:
I think we need to make sure that webpack is going to be doing tree-shaking if we are bundling lodash. Lodash is huge, and if we pull the whole thing, that's going to blow up the size of the bundle by a very large percentage. We need to make sure that only the modules that we are actually using are bundled (just ran into the same exact issue at work, so want to make sure that we verify this before merging).

Reply:
This is only excluding it from running `babel-loader` on the `lodash` files. I do think it's worth taking a look to see if we're including all of `lodash` in our client-side assets here, but this should be the same as it is currently.